external/storm-solr/src/main/java/org/apache/storm/solr/bolt/SolrUpdateBolt.java

```diff
@@ -131,6 +150,21 @@ private void failQueuedTuples(List<Tuple> failedTuples) {
         return queuedTuples;
     }
 
+    @Override
+    public Map<String, Object> getComponentConfiguration() {
```

Comment:
OK. The helper method is `createComponentConfigByTickFreqSecs` in TupleUtils.
And I will make a new PR for other external modules.